.test-infra/jenkins/PrecommitJobBuilder.groovy

```diff
@@ -116,7 +116,7 @@ class PrecommitJobBuilder {
       steps {
         gradle {
           rootBuildScriptDir(commonJobProperties.checkoutDir)
-          tasks(gradleTask)
+          tasks(':installVendoredGrpc', gradleTask)
```

Comment:
Jenkins does not respect this until we tell Jenkins to do so. (That might affect other PRs?)